src/onnx/parse_dequantizelinear.cpp

```diff
@@ -0,0 +1,49 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_dequantizelinear : op_parser<parse_dequantizelinear>
+{
+    std::vector<op_desc> operators() const { return {{"DequantizeLinear"}}; }
+
+    instruction_ref parse(const op_desc& /*opd*/,
+                          const onnx_parser& /*parser*/,
+                          const onnx_parser::node_info& info,
+                          std::vector<instruction_ref> args) const
+    {
+        int axis = 1;
+        if(contains(info.attributes, "axis"))
+            axis = info.attributes.at("axis").i();
+
```

Comment:
We should be able to handle the case of axis to be minus value as:
if (axis < 0) axis += n_rank;

Reply:
fixed